Automattic/jetpack
# Use CompactFormToggle instead of passing a prop that is no longer accepted

We should be using the `CompactFormToggle` component here, rather than passing a prop to `FormToggle` that is no longer accepted after the newest update to dops-components.  

This also required a minor scss change, so tagging for design review as well.  

To test: 
- link to https://github.com/Automattic/dops-components/pull/94
- Toggles should look nice
- Toggles should toggle when clicking the label

This PR can replace #6409, and will close out #6200
